taxonomies/categories.txt

```diff
@@ -34541,8 +34544,25 @@ zh:猪肉餐
 <en:Pork meals
 <en:skewers
 en:pork skewers
-fr:Brochettes de porc, brochette de porc
-nl:Varkensspiesjes
+fr:Brochettes de porc, brochette de porc, échine de porc rôtie, échine de porc au four
+nl:Varkensspiesjes 
+ciqual_food_code:en:28451
+ciqual_food_name:en:Pork, loin, roasted/baked
+ciqual_food_name:fr:Porc, échine, rôtie/cuite au four
+
+<en: Pork meals
+en:Pork ham escalope cooked
+fr:porc cuit, escalope de jambon
```

Comment:
entries like "fr: abc, xyz" will make "xyz" a synonym for "abc", which is not the case here.